.github/workflows/stale-bot.yml

```diff
@@ -1,23 +1,23 @@
 ---
-name: 'Close stale issues and PRs'
+name: Staleness
```

Comment:
⛏️ IMO, the old name was more descriptive. I'd at least call it "Staleness checks".